src/pages/validator/external-chains/centrifuge.mdx

```diff
@@ -296,39 +331,29 @@ sudo systemctl status centrifuge.service
 sudo journalctl -u centrifuge.service -f
 ```
 
-<br></br>
 
 ##### 3. Test your RPC connection
 
-Once your node is fully synced, you can run a cURL request to see the status of your node:
+Once your node is fully synced, you can run a cURL request to see the status of your node, use
+the port you configured in your `/etc/systemd/system/centrifuge.service` file above
 
 ```bash
-curl -H "Content-Type: application/json" -d '{{"id":1, "jsonrpc":"2.0", "method": "eth_syncing", "params":[]}}' localhost:9933
+curl -H "Content-Type: application/json" \
+-d '{{"id":1, "jsonrpc":"2.0", "method": "eth_syncing", "params":[]}}' \
+localhost:9933
 ```
 
 Expected output if node is synced is `{"jsonrpc":"2.0","result":false,"id":1}`
 
 ### Configure `vald`
 
 In order for `vald` to connect to your local node, your `rpc_addr` should be exposed in
-`vald`'s `config.toml`
+vald's `config.toml`.
 
-<tabs>
-<tab-item title="Mainnet">
 ```bash
```

Comment:
the testnet tab is still needed, since testnet connection is called `centrifuge-2`

Reply:
Good catch I don't know why I deleted that

Reply:
@gpmayorga if u add this back, we can merge the PR. and then we can address Milap's other comments in a new branch

Reply:
Just re-added the testnet tab